tox.ini

```diff
@@ -3,7 +3,7 @@ env_list =  # Environments to run when called with no parameters.
     format-{black,ruff,codespell}
     pre-commit
     lint-{black,ruff,mypy,pyright,shellcheck,codespell,docs,yaml}
-    unit-py3.{8,10,11}
+    unit-py3.{10,12}
```

Comment:
Should we also run it against 3.11?

Reply:
`{oldest,newest}` is fine for the default environments.

Reply:
So I left it as is. Thank you!